pkg/cmd/stack/destroy.go

```diff
@@ -176,6 +176,28 @@ func destroyServicesNotInStack(ctx context.Context, spinner *utils.Spinner, s *m
 		spinner.Start()
 	}
 
+	pvcList, err := volumes.List(ctx, s.Namespace, s.GetLabelSelector(), c)
```

Comment:
You probably need to do `s.Volumes = nil` in line 80 of this file to follow the same logic that we do volumes. We might need to for `endpoints`